nbconvert/exporters/templateexporter.py

```diff
@@ -163,6 +163,31 @@ def _template_file_default(self):
     
     #Extension that the template files use.
     template_extension = Unicode(".tpl").tag(config=True, affects_environment=True)
+    
+    exclude_input = Bool(False,
+        help = "This allows you to exclude code cell inputs from all templates if set to True."
```

Comment:
It's not immediately obvious from the names and the help strings how `exclude_input` and `exclude_code` differ.